manifests/0000_50_olm_00-installplans.yaml

```diff
@@ -1,304 +1,307 @@
-          type: string
-        kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-          type: string
-        metadata:
-          type: object
-        spec:
-          description: InstallPlanSpec defines a set of Application resources to be
-            installed
-          type: object
-          required:
-          - approval
-          - approved
-          - clusterServiceVersionNames
-          - generation
-          properties:
-            approval:
-              description: Approval is the user approval policy for an InstallPlan.
-              type: string
-            approved:
-              type: boolean
-            clusterServiceVersionNames:
-              type: array
-              items:
+    singular: installplan
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - description: The first CSV in the list of clusterServiceVersionNames
+      jsonPath: .spec.clusterServiceVersionNames[0]
+      name: CSV
+      type: string
+    - description: The approval mode
+      jsonPath: .spec.approval
+      name: Approval
+      type: string
+    - jsonPath: .spec.approved
+      name: Approved
+      type: boolean
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: InstallPlan defines the installation of a set of operators.
+        type: object
+        required:
+        - metadata
+        - spec
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: InstallPlanSpec defines a set of Application resources to
+              be installed
+            type: object
+            required:
+            - approval
+            - approved
+            - clusterServiceVersionNames
+            - generation
```

Comment:
Should this be required?

Reply:
Isn’t that fixed by the other PR you made?